src/js/view/view.js

```diff
@@ -482,23 +482,32 @@ function View(_api, _model) {
 
     function clickHandlerHelper(api, model, videoLayer) {
         const clickHandler = new ClickHandler(model, videoLayer, { useHover: true });
+        const controls = _model.get('controls');
         clickHandler.on({
             click: () => {
                 _this.trigger(DISPLAY_CLICK);
-                if (_model.get('controls')) {
-                    api.play(reasonInteraction());
+
+                if (_controls) {
+                    if (settingsMenuVisible()) {
```

Comment:
The AC specifically says not to pause the player if the display is clicked and the menu is open.

`When the menu is open, anywhere the the click to pause functionality works should instead close the menu and not pause the player`